warehouse/query-core/src/main/java/datawave/query/transformer/GroupingTransform.java

```diff
@@ -168,17 +155,15 @@ public GroupingTransform(BaseQueryLogic<Entry<Key,Value>> logic, Collection<Stri
         
         // Handle if the current page has exceeded its execution timeout, but there are still more results to return
         // This must be done BEFORE popping documents from the document stack.
-        if (allowLongRunningQuery && this.queryExecutionForPageStartTime > 0) {
-            long elapsedExecutionTimeForCurrentPage = System.currentTimeMillis() - this.queryExecutionForPageStartTime;
-            if (elapsedExecutionTimeForCurrentPage > this.queryExecutionForPageTimeout) {
-                // Reset the queryExecutionForPageStartTime and clear the documents list so that it doesn't contain
-                // duplicates. Then return an empty document with the intermediate result flag set to true
-                this.queryExecutionForPageStartTime = System.currentTimeMillis();
-                documents.clear();
-                Document intermediateResult = new Document();
-                intermediateResult.setIsIntermediateResult(true);
-                return Maps.immutableEntry(new Key(), intermediateResult);
-            }
+        // The first page queryExecutionForPageStartTime will be 0
```

Comment:
The first queryExecutionForPageStartTime isn't 0, is it?  It's set to System.currentTimeMillis() at the beginning of each next call, right?  Guessing this is a holdover from a previous implementation?

Reply:
Yes, you're right. I meant to remove that comment.  Will remove it.